pylint/checkers/base/basic_checker.py

```diff
@@ -671,15 +671,23 @@ def _is_terminating_func(node: nodes.Call) -> bool:
 
         inferred = utils.safe_infer(node.func)
 
-        if not inferred and isinstance(node.func, nodes.Attribute) and node.func.attrname == "_exit":
+        if (
```

Comment:
@clavedeluna You might want to install `pre-commit` locally. Although it runs automatically in CI, all the CI jobs that start on the commit that is then rejected by `pre-commit` (in this case https://github.com/PyCQA/pylint/pull/7520/commits/59feeaeae8fdebacd878b197c8ae6a2c995e98b5) do take up a "runner" and some minutes of our CI. When there are multiple people working on various PRs this can block our CI a bit, especially since the Primer jobs block runners for 30+ min. Running `pre-commit` locally prevents taking up those initial runners.

Reply:
I was just looking into this bc the number of pre-commit commits is getting annoying. I can run `black` locally, but what's the procedure for using .pre-commit-comfig.yaml locally?

Reply:
Check out https://pre-commit.com 😄